src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  switch (days) {
+    case 7:
+      return 7 * PRICE - LONG_TERM_DISCOUNT;
+    case 6:
+      return 6 * PRICE - SHORT_TERM_DISCOUNT;
+    case 5:
+      return 5 * PRICE - SHORT_TERM_DISCOUNT;
+    case 4:
+      return 4 * PRICE - SHORT_TERM_DISCOUNT;
+    case 3:
+      return 3 * PRICE - SHORT_TERM_DISCOUNT;
+    case 2:
+      return 2 * PRICE;
+    case 1:
+      return 1 * PRICE;
+  }
```

Comment:
Using a switch statement for this logic isn't the best approach as it creates unnecessary repetition. Consider using if statements to check for the discount conditions. Also, the current logic only provides a discount for exactly 7 days, but the requirement is for 7 or more days. Similarly, the discount for 3 or more days is not handled correctly; it should apply to 3, 4, 5, and 6 days as well.